src/server/DBMetadata/classes/processedField.js

```diff
@@ -51,7 +51,7 @@ class ProcessedField {
 		const { foreign_table_name, foreign_column_name } = tblCol;
 
 		// Active relationship assignment (primary key table defined before foreign key table)
-		if (lookup.hasOwnProperty(foreign_table_name)) {
+		if (Object.prototype.hasOwnProperty.call(foreign_table_name)) {
```

Comment:
This is unnecessarily verbose. Technically since all objects created inherit from Objects prototype, you can just use lookup.hasOwnProperty(foreign_table_name). There are cases where objects can be created that don't inherit from Objects prototype, but you have to actively try to do it.